decentraland/unity-explorer
# fix: profile requests spam

## What does this PR change?

Fixes #1073 #1074 

The multiplayer was requesting the same profiles many times before they were resolved. Also there was a cache miss on profiles versioning.

## How to test the changes?

1. Multiple users should enter to the same zone. The more the better.
2. Move around, play emotes, do stuff. Check that the avatar propagation works as expected
3. Change wearables and/or emotes
4. Check that the avatar propagation works as expected
5. Monitor realm metrics, check that the profile request amount and entities/active request amount is normal

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
